.reaction/waitForReplica.js

```diff
@@ -0,0 +1,161 @@
+/*
+/opt/reaction/src/imports/node-app/core/util/waitReplica.js
+   3:1   warning  Missing JSDoc comment                                require-jsdoc
+   8:1   warning  Missing JSDoc comment                                require-jsdoc
+  12:1   warning  Missing JSDoc comment                                require-jsdoc
+  33:1   warning  Missing JSDoc comment                                require-jsdoc
+  34:22  error    Expected parentheses around arrow function argument  arrow-parens
+  37:1   warning  Missing JSDoc comment                                require-jsdoc
+  39:3   warning  Missing JSDoc comment                                require-jsdoc
+  65:5   error    Unary operator '++' used                             no-plusplus
+  67:16  error    Unexpected `await` inside a loop                     no-await-in-loop
+  70:7   error    Unexpected `await` inside a loop                     no-await-in-loop
+  78:5   error    Unexpected `await` inside a loop                     no-await-in-loop
+
+✖ 11 problems (5 errors, 6 warnings)
+*/
```

Comment:
Did you intend to leave this comment in here?

Reply:
Nope, just notes to self. Good catch. My eslint editor integration isn't set up yet for this environment. Fixed with an amend and force push.